### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
First Commit

### CODE REVIEW

File: amazon-cloudwatch-logs/constant.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+""" Copyright start
```
Comment (cs-prerna-joshi) [inline amazon-cloudwatch-logs/constant.py]: rename file as constants.py

File: amazon-cloudwatch-logs/info.json [hunk 2]
```diff
@@ -0,0 +1,664 @@
+{
+  "name": "amazon-cloudwatch-logs",
+  "version": "1.0.0",
+  "label": "Amazon CloudWatch Logs",
+  "description": "Amazon CloudWatch Logs enables you to monitor, store, and access your system, application, and custom log files. This connector facilitates the automate operations related to the log group, log streams and metrics.",
+  "publisher": "Community",
+  "cs_approved": false,
+  "cs_compatible": true,
+  "category": "AWS Services",
+  "icon_small_name": "cyops_small.png",
```
Comment (cs-prerna-joshi) [inline amazon-cloudwatch-logs/info.json:10]: please add correct logo

File: amazon-cloudwatch-logs/info.json [hunk 3]
```diff
@@ -0,0 +1,664 @@
...
+  "configuration": {
+    "fields": [
+      {
+        "title": "AWS Access Key",
+        "type": "password",
+        "name": "aws_access_key_id",
+        "required": true,
+        "visible": true,
+        "editable": true,
+        "value": null,
```
Comment (cs-prerna-joshi) [inline amazon-cloudwatch-logs/info.json]:
```suggestion
        "value": "",
```

File: amazon-cloudwatch-logs/info.json [hunk 4]
```diff
@@ -0,0 +1,664 @@
...
+        "description": "The access key for users AWS account"
+      },
+      {
+        "title": "AWS Secret Key",
+        "type": "password",
+        "name": "aws_secret_access_key",
+        "required": true,
+        "visible": true,
+        "editable": true,
+        "value": null,
```
Comment (cs-prerna-joshi) [inline amazon-cloudwatch-logs/info.json]:
```suggestion
        "value": "",
```

File: amazon-cloudwatch-logs/info.json [hunk 5]
```diff
@@ -0,0 +1,664 @@
...
+        },
+        {
+          "title": "Start Time",
+          "description": "The start of the time range for which log events to be fetched.",
+          "type": "datetime",
+          "name": "startTime",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "value": null
```
Comment (cs-prerna-joshi) [inline amazon-cloudwatch-logs/info.json]:
```suggestion
          "value": ""
```

File: amazon-cloudwatch-logs/info.json [hunk 6]
```diff
@@ -0,0 +1,664 @@
...
+        },
+        {
+          "title": "End Time",
+          "description": "The end of the time range for which log events to be fetched.",
+          "type": "datetime",
+          "name": "endTime",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "value": null
```
Comment (cs-prerna-joshi) [inline amazon-cloudwatch-logs/info.json]:
```suggestion
          "value": ""
```

Comment (cs-rohit-khune) [reply amazon-cloudwatch-logs/info.json]: Done

File: amazon-cloudwatch-logs/info.json (hunk 5 above)
Comment (cs-rohit-khune) [reply amazon-cloudwatch-logs/info.json]: Done

File: amazon-cloudwatch-logs/info.json (hunk 4 above)
Comment (cs-rohit-khune) [reply amazon-cloudwatch-logs/info.json]: Done

File: amazon-cloudwatch-logs/info.json (hunk 3 above)
Comment (cs-rohit-khune) [reply amazon-cloudwatch-logs/info.json]: Done

File: amazon-cloudwatch-logs/info.json (hunk 2 above)
Comment (cs-rohit-khune) [reply amazon-cloudwatch-logs/info.json:10]: Done

File: amazon-cloudwatch-logs/constant.py (hunk 1 above)
Comment (cs-rohit-khune) [reply amazon-cloudwatch-logs/constant.py]: Done

**Pull request merged**